docs/src/templates/pages/styleguide/ui-templates/menu/menu-delivery.hbs

```diff
@@ -0,0 +1,27 @@
+---
+title: Menu UI Template
+docs: true
+
+layout: site-frame
+styleguide: true
+bodyClass: o-body--bgWhite--belowMid
+---
+
+<div class="c-pageBanner c-pageBanner--negativeBottom--aboveMid">
+    <picture>
+        <source class="c-pageBanner-img" media="(min-width: 860px)" srcset="{{baseUrl}}/assets/img/examples/kebabs-1-uk_1440x617.jpg" data-ft="restaurant-level-image--wide">
```

Comment:
Is it worth renaming these image files to be a bit more generic (could just be `kebabs--wide.jpg`, `kebabs--mid` & `kebabs--narrow`?)

Reply:
Done.